packages/@aws-cdk/aws-events-targets/lib/log-group.ts

```diff
@@ -0,0 +1,52 @@
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as logs from '@aws-cdk/aws-logs';
+import * as cdk from '@aws-cdk/core';
+import { LogGroupResourcePolicy } from './log-group-resource-policy';
+/**
+ * Customize the CloudWatch LogGroup Event Target
+ */
+export interface LogGroupProps {
+  /**
+   * The event to send to the CloudWatch LogGroup
+   *
+   * This will be the event logged into the CloudWatch LogGroup
+   *
+   * @default - the entire EventBridge event
+   */
+  readonly event?: events.RuleTargetInput;
+}
+
+/**
+ * Use an AWS CloudWatch LogGroup as an event rule target.
+ */
+export class LogGroup implements events.IRuleTarget {
+  constructor(private readonly logGroup: logs.ILogGroup, private readonly props: LogGroupProps = {}) {}
+
+  /**
+   * Returns a RuleTarget that can be used to log an event into a CloudWatch LogGroup
+   */
+  public bind(_rule: events.IRule, _id?: string): events.RuleTargetConfig {
+    // Use a custom resource to set the log group resource policy since it is not supported by CDK and cfn.
+    // https://github.com/aws/aws-cdk/issues/5343
+    const resourcePolicyId = `EventsLogGroupPolicy${_rule.node.uniqueId}`;
+    if (!this.logGroup.node.tryFindChild(resourcePolicyId)) {
+      new LogGroupResourcePolicy(cdk.Stack.of(this.logGroup), resourcePolicyId, {
```

Comment:
does this approach work with an imported log group - i.e. one that was passed in by calling `fromLogGroupName` or `fromLogGroupArn`?

Reply:
`ILogGroup` extends `IResource`. So it should be fine. Just added another integration test for that.